scan/src/components/PieChart.re

```diff
@@ -0,0 +1,63 @@
+module Styles = {
+  open Css;
+
+  let pie = (size, color) =>
+    style([
+      overflow(`hidden),
+      display(`flex),
+      position(`relative),
+      width(`px(size)),
+      height(`px(size)),
+      borderRadius(`percent(100.)),
+      backgroundColor(color),
+    ]);
+
+  let segment = (offset, asize, color, isHidden) =>
+    style([
+      visibility(isHidden ? `hidden : `visible),
+      overflow(`hidden),
+      width(`percent(100.)),
+      height(`percent(100.)),
+      position(`absolute),
+      transforms([
+        `translate((`zero, `percent(-50.))),
+        `rotate(`deg(90.)),
+        `rotate(`deg(offset)),
+      ]),
+      transformOrigin(`percent(50.), `percent(100.)),
+      before([
+        contentRule(`text("")),
+        width(`percent(100.)),
+        height(`percent(100.)),
+        position(`absolute),
+        transforms([`translate((`zero, `percent(100.))), `rotate(`deg(asize))]),
+        transformOrigin(`percent(50.), `percent(0.)),
+        backgroundColor(color),
+      ]),
+    ]);
+};
+
+let renderSegment = (offset, asize, color) =>
+  <>
+    <div className={Styles.segment(offset, asize <= 180.0 ? asize : 180.0, color, false)} />
+    <div
+      className={Styles.segment(
+        offset +. 180.0,
+        asize <= 180.0 ? 0.0 : asize -. 180.0,
+        color,
+        asize <= 180.0,
+      )}
+    />
+  </>;
+
+[@react.component]
+let make = (~size=187, ~availableBalance=500., ~balanceAtStake=60., ~reward=20.) => {
```

Comment:
Let's not have default value here. If we want to mock something, we do in AccountIndexPage